charts/platform-system/templates/flux/istio/_istio.tpl

```diff
@@ -0,0 +1,65 @@
+{{ define "platform-system.istio.repository.name" }}
+{{- printf "%s-%s" .Release.Name .Values.apps.istio.chart.source.repository | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end -}}
+
+{{/*
+  Istio base templates
+*/}}
+
+{{ define "platform-system.istio.istio-base.release.name" }}
```

Comment:
`istioBase`, but do we really need to separate functions for something? Just add an optional parameter for suffix of `-base`?